hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,5 @@
+RELEASE_TYPE: patch
+
+Read environment variable HYPOTHESIS_NO_ENTRYPOINTS to
+optionally skip loading of third-party plugin entrypoints.
```

Comment:
```suggestion
If the :envvar:`HYPOTHESIS_NO_PLUGINS` environment variable is set, we'll avoid 
:ref:`loading plugins <>` such as `the old Pydantic integration 
<https://docs.pydantic.dev/latest/integrations/hypothesis/>`__ or 
`HypoFuzz' CLI options <https://hypofuzz.com/docs/quickstart.html#running-hypothesis-fuzz>`__.

This is probably only useful for our own self-tests, but documented in case it might
help narrow down any particularly weird bugs in complex environments.
```

Note that `loading plugins` will need a cross-ref inserted in our own docs (and potentially a label added - I'm not sure); and that I've renamed the envvar.